ftplugin/python/pytest.vim

```diff
@@ -896,7 +896,7 @@ function! s:ThisMethod(verbose, ...)
       return
     endif
     if ((a:1 == '--pdb') || (a:1 == '-s'))
-        call s:Pdb(path, a:1)
+        call s:Pdb(path, a:1, m_name)
```

Comment:
you can re-use the `parametrized_flag` which would make it easier to detect later (vs. using `m_name` and checking again if the method is parametrized)